ksrates/fc_configfile.py

```diff
@@ -214,21 +239,21 @@ def get_gff_dict(self, warn_empty_dict=True):
 
     def get_gff_name(self, gff_dict, species):
         """
-        Gets the path to the GFF file of the species of interest from the dictionary.
+        Gets the path to the GFF file of the focal species from the dictionary.
         If the value is an empty string, it applies the fallback filename "species + .gff".
 
-        :param fasta_dict: Python dictionary that associates each informal species name to the path of its FASTA file 
-        :param species: the informal name of the species of interest
+        :param gff_dict: Python dictionary that associates the focal species informal name to the path of its GFF file 
```

Comment:
Why is this necessary? It's always for the focal species.

Reply:
I see what you mean, alright I'll change it :)